flax/core/lift.py

```diff
@@ -215,84 +216,48 @@ def repack(inner_scope_tree):
 id_fn = lambda x: x
 
 
-def transform(
+def map_variables(
     fn: Callable[..., Any],
-    target: CollectionFilter,
-    trans_in_fn: Callable[..., Any] = id_fn,
-    trans_out_fn: Callable[..., Any] = id_fn,
+    mapped_collections: CollectionFilter,
+    map_in_fn: Callable[..., Any] = id_fn,
+    map_out_fn: Callable[..., Any] = id_fn,
     init: bool = False, mutable: bool = False,
     rngs: PRNGSequenceFilter = True, variables: CollectionFilter = True):
-  """Locally transform Variables inside a scope.
+  """Locally map Variables inside a scope.
 
   Args:
     fn: the function to be transformed.
     target: the collection(s) to be transformed.
```

Comment:
target -> mapped_collections?

here and below I see both `target` and `mapped_collections` used but clearly only one makes sense.